apps/explorer/components/Faucet/FaucetFundForm.tsx

```diff
@@ -1,94 +1,103 @@
 import {
-  FormFieldFormik,
-  FormSubmitButtonFormik,
+  Button,
+  ConfigFields,
+  FieldSiacoin,
+  FieldText,
   Paragraph,
+  triggerErrorToast,
   triggerSuccessToast,
 } from '@siafoundation/design-system'
 import { toHastings } from '@siafoundation/units'
 import BigNumber from 'bignumber.js'
-import { useFormik } from 'formik'
-import * as Yup from 'yup'
 import { networkName } from '../../config'
 import { useFaucetFund } from '../../hooks/useFaucetFund'
+import { useForm } from 'react-hook-form'
 
-const initialValues = {
+const defaultValues = {
   address: '',
   amount: undefined as BigNumber | undefined,
 }
 
-const validationSchema = Yup.object().shape({
-  address: Yup.string().required('Required'),
-  amount: Yup.string()
-    .required('Required')
-    .test(
-      'greater than zero',
-      'Must be greater than 0 SC',
-      (val) => !new BigNumber(val || 0).isZero()
-    )
-    .test('less than 1000', 'Must be 50,000 SC or less', (val) =>
-      new BigNumber(val || 0).lte(50_000)
-    ),
-})
+const fields: ConfigFields<typeof defaultValues, never> = {
+  address: {
+    type: 'text',
+    title: 'Address',
+    placeholder: '020d48a57bc0...',
+    validation: {
+      required: 'An address is required.',
+    },
+  },
+  amount: {
+    type: 'siacoin',
+    title: 'Amount',
+    showFiat: false,
+    validation: {
+      required: 'An amount is required',
+      validate: {
+        min: (value: BigNumber) =>
+          new BigNumber(value || 0).gte(1) || 'Amount must be greater than 0',
+        max: (value: BigNumber) =>
```

Comment:
I think this would be true for the `|| 0`, the above syntax would fix that issue.